src/screens/UserPortal/Chat/Chat.spec.tsx

```
@@ -4280,7 +4281,126 @@ describe('Testing Chat Screen [User Portal]', () => {
     ...GROUP_CHAT_LIST_QUERY_MOCK,
     ...MARK_CHAT_MESSAGES_AS_READ_MOCK,
   ];
+  beforeEach(() => {
+    vi.clearAllMocks();
+  });
```

Biome (1.9.4)

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)